lib/util/include/hse_util/mutex.h

```diff
@@ -64,6 +64,10 @@
  * descheduling.
  */
 
+#include <errno.h>
+#include <pthread.h>
+#include <stdlib.h>
+
```

Comment:
Not clear what in mutex.h needs errno.h nor stdlib.h.  But it always should have had pthread.!

Reply:
There is an EBUSY reference and abort() reference.